workspaces/cli-server/src/config.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  createCliConfig,
+  Env,
+  CliConfigObject,
+  isEnvTrue,
+  PackageJson,
+} from '@useoptic/cli-shared';
+const pkg = require('../package.json') as PackageJson;
+
+const cliConfig = createCliConfig(pkg, process.env);
+
+export default {
+  ...cliConfig,
+
+  errors: {
+    sentry: cliConfig.env.production &&
```

Comment:
This does make it slightly more difficult to enable sentry during development which I did find helpful. Will you be making the corresponding changes to cli-scripts?

Reply:
I've only ever found enabling sentry locally helpful in implementing the integration with sentry, and _have_ found it super annoying to find development errors to be tracked because it was accidentally enabled, so I optimised with that hard-won experience in mind 😅.

I actually don't think DSN and such has to come from the environment either, as it's often 1:1 mapped to the client, but it's there so 🤷‍♂️. Maybe if we want to clean up the .env files at some point.

And yes, going to apply this to `cli-scripts` as well! Forgot to mention that this is draft because I wanted to agree on the approach first, before spending the time to applying it fully.